flowise-embed/src/utils/fileClassificationUtils.ts

```diff
@@ -70,27 +70,27 @@ export const checklistCommercialInvoice = `
 • Número do documento
 • Nome do documento
 • Data do documento
-• Dados do Importador - também chamado de Consignee, Importer, Ship To (Razão social, endereço e CNPJ, CEP); Em casos em que não está esplicitamente indicado, os primeiros dados que constam no documento são considerados como dados do importador.
-• Dados do Adquirente ou Encomendante - também chamado de Notify, Buyer, Sold to, Encomendante, Bill to, Notify Party (Razão social, endereço, CNPJ, CEP)
+• Dados do Importador - também chamado de Consignee, Importer, Ship To, traga os valores desse campo; Em casos em que não está explicitamente indicado, os primeiros dados que constam no documento são considerados como dados do importador.
```

Comment:
Acredito que dê pra deixar mais clara a parte do "traga os valores desse campo" e separar com outro tipo de pontuação. Talvez algo como: `também chamado de ..., Ship To; Obtenha todos os dados relacionados ao importador; Em casos...` - não gostei tanto do meu exemplo, ainda dá pra melhorar mais, mas o principal é a ideia da pontuação